### DISCUSSION TITLE
Take over the somewhat abandoned shc organization name

### DISCUSSION THREAD

**Discussion by suntong** [community/community#136303] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, 

I've researched the topic before posting, and I know that I have less than 50/50 percent chances of wining my case, so this is just a kind request/discussion to the github admin that we intend to take over the somewhat abandoned "**shc**" organization name.

### What

First of all, what shc we intend to use as the github organization? It's originated at 
https://www.datsi.fi.upm.es/~frosal/sources/shc.html, and went through different homes, with the last place being on github at https://github.com/neurobin/shc/. 

It's a well-known tools to the *nix world, and it has been used by people all around the world for over 20 years. 

I am the DM of it, and I maintain the Debian version at
https://salsa.debian.org/debian/shc

### Why

As said before, shc has gone through different homes, and even for the latest place the author [has not been responding on this project for about a year](https://github.com/neurobin/shc/pull/153#issuecomment-2295238213). That's not a big deal until _there is a fatal issue which is causing it to have been removed from the Debian testing repo_.

The PR of fixing the issue has been sent but got no responses. 

So a small group of volunteers chipped in and bring the project to the correct stage, taking along all the PR that had been sitting in the repo for quite a while. 

Now we need to find a better home for it, to share the maintaining responsibilities, including me, who want it to have a permanent home to be documented at
https://salsa.debian.org/debian/shc/-/blob/debian/master/debian/control?ref_type=heads

### The road block

The github.com/shc organization was

- first [created in 2022 and only have few (private) commits](https://github.com/shc?tab=overview&from=2022-12-01&to=2022-12-31)
- and in the next year, [2023, there were only 4 private commits](https://github.com/shc?tab=overview&from=2023-12-01&to=2023-12-31)
- and [total silent this year](https://github.com/shc?tab=overview&from=2024-08-01&to=2024-08-19)

Most importantly 

- it's _**used by a single user**_, so it literately is not an organization.
- Thus, it being not an organization somewhat violates the [GitHub Name Squatting Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#name-squatting-policy)

### The petition

So we're kindly petitioning to rename the dormant shc organization to other name, like org-shc, as _"Accounts violating this name squatting policy may be removed or renamed without notice"_.

Please consider. 

Thanks a lot, in helping our Free Software Endeavor

**Comment by ebndev** [comment, marked as answer]
Hi @suntong,

Unfortunately, we are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
